org.nitrokey.app.yml

```diff
@@ -0,0 +1,59 @@
+app-id: org.nitrokey.app
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: nitrokey-app
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --filesystem=host
+  - --device=all
+modules:
+  - name: libusb
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://github.com/libusb/libusb/archive/v1.0.24.tar.gz
+        sha256: b7724c272dfc5713dce88ff717efd60f021ca5b7c8e30f08ebb2c42d2eea08ae
+  - name: hidapi
+    buildsystem: autotools
+    sources:
+    - type: archive
+      url: https://github.com/libusb/hidapi/archive/hidapi-0.10.1.tar.gz
+      sha256: f71dd8a1f46979c17ee521bc2117573872bbf040f8a4750e492271fc141f2644
+  - name: libnitrokey
+    buildsystem: cmake
+    sources:
+    - type: archive
+      url: https://github.com/Nitrokey/libnitrokey/archive/v3.6.tar.gz
+      sha256: 89bc89ef14597186b57d82ebf696e9318528dad2a748704145d9e3320084dde7
+  - name: Catch2
+    buildsystem: cmake
+    builddir: true
+    sources: 
+    - type: archive
+      url: https://github.com/catchorg/Catch2/archive/v2.13.4.tar.gz
+      sha256: e7eb70b3d0ac2ed7dcf14563ad808740c29e628edde99e973adad373a2b5e4df
+  - name: cppcodec
+    buildsystem: cmake
+    sources:
+    - type: archive
+      url: https://github.com/tplgy/cppcodec/archive/v0.2.tar.gz
+      sha256: 0edaea2a9d9709d456aa99a1c3e17812ed130f9ef2b5c2d152c230a5cbc5c482
+  - name: nitrokey-app
+    buildsystem: cmake
+    sources:
+      - type: archive
+        url: https://github.com/Nitrokey/nitrokey-app/releases/download/v1.4.2/nitrokey-app-v1.4.2.tar.gz
+        sha256: 965c05409c0fb534b8092eb66a54bafd4cb1a2e50f15cf197f6527ae14747634
+  - name: nitrokey-meta
+    buildsystem: simple
+    build-commands:
+      - "install -Dm644 ${FLATPAK_ID}.appdata.xml /app/share/metainfo/${FLATPAK_ID}.metainfo.xml"
+      - "mv /app/share/icons/hicolor/scalable/apps/nitrokey-app.svg /app/share/icons/hicolor/scalable/apps/${FLATPAK_ID}.svg"
+      - "mv /app/share/applications/nitrokey-app.desktop /app/share/applications/${FLATPAK_ID}.svg"
```

Comment:
Invalid target file extension on this line

Reply:
oops ;)